docs/source/guides/tools/rpakpilotmodding.rst

```diff
@@ -0,0 +1,245 @@
+-----
+
+- Select ``Edit`` then ``Preferences``
+- Press ``Install`` at the top right
+- Locate the ``Blender Source Tools.zip`` zip file and select it (Make sure to enable it too)
+- Select ``File`` at the top left, import, then ``Source Engine``
+- Locate and select the decompiled pilot model
+  
+Make any changes you want. This is the customization part.
+
+- The head of your model should be separate from the rest of the body. 
+- You can use the bisect tool in edit mode to separate the head if it isn’t already.
+- Make sure the body and head of your model match the names of the pilot model
+
+Drag your model’s head and body into the correct collection. 
+Delete the pilot models head and body after doing so. 
+Renaming and putting your model in the correct collection.
+
+- Rig the model to match the bones of the armature that came with the pilot model
+- Create an armature modifier for your model’s body and head
+
+.. image:: /_static/importpilotmodel/ss0-pilotBlenderArmature.png
+   :align: center
+   :class: screenshot
+
+
+Attach the model’s body and head to the armature.
+The video does not have the model properly aligned with the armature. Please make sure your model is aligned with the armature before attaching the model to it.
+A properly rigged model should look like this:
+
+.. image:: /_static/importpilotmodel/ss1-pilotBlender.png
+   :align: center
+   :class: screenshot
+
+If you have multiple textures on your model, you would create several materials that match the name of the textures for the pilot you are modifying.
+If you only have one texture for your model, create one material for your entire model.
+Use LegionPlus to view paths for materials. Scroll below a little bit to learn how to use Legion+.
+
+.. image:: /_static/importpilotmodel/ss2-pilotBlenderMatl.png
+   :align: center
+   :class: screenshot
+
+Once you have made all the changes you wanted to, select ``Scene Properties``, ``Source Engine Export``, then set the export format to ``SMD`` and set your export path.
+
+.. image:: /_static/importpilotmodel/ss3-BlenderQcModifier.png
+   :align: center
+   :class: screenshot
+
+Copy the qc file to the same location where you exported the model.
+
+Modifying the POV model
+^^^^^
+
+- Select ``File`` in Blender, then select ``Save Copy``
+- Choose any name, save the copy, then open the copy
+- Delete everything except for the body, the body collection, and the skeleton
+- Add ``pov_`` to the beginning of your body model and armature
+- Repeat the process of exporting the model with ``Source Engine Export``
+
+Using Legion+ to view paths
+^^^^^
+
+Open `Legion+`_
+-----
+
+Select ``Load File``
+Follow the directory that matches your launcher.
+
+1. Steam: ``Steam\steamapps\common\Titanfall2\r2\paks\Win64\common.rpak`` or
+2. Origin: ``Origin Games\Titanfall2\r2\paks\Win64\common.rpak``
+
+Type the file name of the pilot into the search bar to view **texture and material paths**.
+View file paths to create paths in RePak
```

Comment:
This should advise to turn on "Show full asset paths" in Legion+ settings, which I believe is off by default

Reply:
I noticed that but this was just an .rst translation so I kept it "as is".  Considering the amount of adjustments and rewriting, I will now add your suggestion.